plugins/orchestrator/src/components/ExecuteWorkflowPage/ExecuteWorkflowPage.tsx

```diff
@@ -27,154 +30,187 @@ import {
   executeWorkflowWithBusinessKeyRouteRef,
   workflowInstanceRouteRef,
 } from '../../routes';
-import { BaseOrchestratorPage } from '../BaseOrchestratorPage/BaseOrchestratorPage';
-import { OrchestratorSupportButton } from '../OrchestratorSupportButton/OrchestratorSupportButton';
-import { WorkflowDialog } from '../WorkflowDialog';
-import { EditorViewKind } from '../WorkflowEditor';
-import { TitleFieldTemplate } from './TitleFieldTemplate';
-
-const WrappedForm = withTheme(require('@rjsf/material-ui-v5').Theme);
+import { getErrorObject } from '../../utils/error';
+import { BaseOrchestratorPage } from '../next/BaseOrchestratorPage';
+import SubmitBtn from '../SubmitBtn/SubmitBtn';
+import StepperForm from './StepperForm';
 
-interface ExecuteWorkflowPageProps {
+export interface ExecuteWorkflowPageProps {
   initialState?: Record<string, JsonValue>;
 }
 
-export const ExecuteWorkflowPage = (props: ExecuteWorkflowPageProps) => {
+const JsonTextAreaForm = ({
+  isExecuting,
+  handleExecute,
+}: {
+  isExecuting: boolean;
+  handleExecute: (
+    getParameters: () => Record<string, JsonValue>,
+  ) => Promise<void>;
+}) => {
+  const [jsonText, setJsonText] = React.useState('');
+  const theme = useTheme();
+  const getParameters = (): Record<string, JsonValue> => {
+    if (!jsonText) {
+      return {};
+    }
+    const parameters = JSON.parse(jsonText);
+    return parameters as Record<string, JsonValue>;
+  };
+
+  return (
+    <Grid container spacing={2}>
+      <Grid item xs={12}>
+        <Alert severity="info" style={{ width: '100%' }}>
+          <AlertTitle>
+            Couldn't find a valid JSON schema to display the input form.
+          </AlertTitle>
+          If you want to use a form to start the workflow, please provide a
+          valid JSON schema in the dataInputSchema property of your workflow
+          definition file. Alternatively, you can type below the input data in
+          JSON format.
+        </Alert>
+      </Grid>
+      <Grid item xs={12}>
+        <Box style={{ border: `1px solid ${theme.palette.border}` }}>
```

Comment:
`useStyles` instead?

Reply:
@caponetto I'm not sure it is worth it in this case. This is just a simple override, a one-off and is well encapsulated (inline).
Using `useStyles` requires calling `makeStyles` and then going over the entire ceremony (which allocates more objects in memory...) of assigning `classNames`. With this implementation the border will be already in sync with the theme.